pkg/resources/account_grant_acceptance_test.go

```diff
@@ -37,3 +37,33 @@ resource "snowflake_account_grant" "test" {
 }
 `, role)
 }
+
+func TestAccAccountGrant_grant_manged_task(t *testing.T) {
```

Comment:
spelling

Reply:
updated to TestAcc_AccountGrantManagedTask